vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -9,19 +9,30 @@ if (-not ($validContainerRuntimes -contains $containerRuntime)) {
 }
 
 $global:windowsSKU = $env:WindowsSKU
-$validSKU = @("2019", "2019-containerd")
+$validSKU = @("2019", "2019-containerd", "2022-containerd")
 if (-not ($validSKU -contains $windowsSKU)) {
     throw "Unsupported windows image SKU: $windowsSKU"
 }
 
 # Windows Server 2019 update history can be found at https://support.microsoft.com/en-us/help/4464619
 # then you can get download links by searching for specific KBs at http://www.catalog.update.microsoft.com/home.aspx
 #
+# Windows Server 2019 update history can be found at # TODO
```

Comment:
Do you mean `Windows Server 2022 update history can be found at`?

Reply:
https://support.microsoft.com/en-gb/topic/windows-server-2022-update-history-e1caa597-00c5-4ab9-9f3e-8212fe80b2ee

Reply:
Updated. Thank you.